creator-node/test/tracks.js

```diff
@@ -445,45 +448,31 @@ describe('test /track_content with actual ipfsClient', function () {
     let storagePath = config.get('storagePath')
     storagePath = storagePath.slice(0, 1) === '/' ? '.' + storagePath : storagePath
 
-    // check if track UUID dir exists
+    // Wait before checking disk state since disk pruning action is made as a non-blocking call and may
+    //  complete after /track_content route returns
+    await utils.timeout(2000)
+
+    // Ensure original trackUUID dir has been pruned and does not exist
     const originalTrackUUID = resp.body.source_file.split('.').slice(0, -1).join('.') // remove extension
     const originalTrackUUIDPath = path.join(storagePath, originalTrackUUID)
-    assert.ok(fs.existsSync(originalTrackUUIDPath))
-
-    // check that the track UUID dir contains the transcoded copy
-    const transcodedTrackPath = path.join(originalTrackUUIDPath, originalTrackUUID + '-dl.mp3')
-    assert.ok(fs.existsSync(transcodedTrackPath))
+    assert.ok(!fs.existsSync(originalTrackUUIDPath))
 
     // check that the generated transcoded track is the same as the transcoded track in /tests
     const transcodedTrackAssetPath = path.join(__dirname, 'testTranscoded320Track.mp3')
     const transcodedTrackAssetBuf = fs.readFileSync(transcodedTrackAssetPath)
+    const transcodedTrackPath = path.join(storagePath, resp.body.data.transcodedTrackCID)
     const transcodedTrackTestBuf = fs.readFileSync(transcodedTrackPath)
     assert.deepStrictEqual(transcodedTrackAssetBuf.compare(transcodedTrackTestBuf), 0)
 
-    // check that the track UUID dir contains the source file
```

Comment:
this probably makes the segments in the test/test-segments/folder redundant.  we can delete those too

Reply:
would actually be good to get the segment CIDs and compare each one on disk to these for equality, i'll add a TODO in this test

Reply:
if we could do this as part of this pr that would be 💯 . ultimate peace of mind that the code moves the segments to the storage path and it's byte for byte the same

Reply:
added this test